api/src/main/java/org/pmiops/workbench/reporting/insertion/UserColumnValueExtractor.java

```diff
@@ -22,13 +22,10 @@
       u -> toInsertRowString(u.getComplianceTrainingCompletionTime())),
   CONTACT_EMAIL("contact_email", ReportingUser::getContactEmail),
   CREATION_TIME("creation_time", u -> toInsertRowString(u.getCreationTime())),
-  DATA_USE_AGREEMENT_BYPASS_TIME(
-      "data_use_agreement_bypass_time", u -> toInsertRowString(u.getDataUseAgreementBypassTime())),
-  DATA_USE_AGREEMENT_COMPLETION_TIME(
-      "data_use_agreement_completion_time",
-      u -> toInsertRowString(u.getDataUseAgreementCompletionTime())),
-  DATA_USE_AGREEMENT_SIGNED_VERSION(
-      "data_use_agreement_signed_version", ReportingUser::getDataUseAgreementSignedVersion),
+  DUCC_BYPASS_TIME("data_use_agreement_bypass_time", u -> toInsertRowString(u.getDuccBypassTime())),
```

Comment:
can you please remind me why the first argument is still data_use_agreeement rather than DUCC

Reply:
That needs to match the reporting DB table, which I have not changed.  I am not sure that it's worth the effort to do that.